README.md

```diff
@@ -1,10 +1,11 @@
-# DAOStack Alchemy - Creating DAOs for an Emergent Future
+# DAOstack Alchemy - Creating DAOs for an Emergent Future
```

Comment:
Not sure about the word emergent here

Reply:
I like it because it reflects a fact in a new and useful way of thinking that is very much part of the epistemology that has led to the existence of DAOstack.  And it is prosaic.

Reply:
I think it is better we avoid it for now.